nb_conda/handlers.py

```diff
@@ -259,7 +259,7 @@ def get(self):
 
 # there is almost no text that is invalid, but no hyphens up front, please
 # neither all these suspicious but valid caracthers...
-_env_regex = r"(?P<env>[^/&+$?@<>%*-][^/&+$?@<>%*]*)$"
+_env_regex = r"(?P<env>[^/&+$?@<>%*-][^/&+$?@<>%*]*)"
```

Comment:
Well, in JS is the same, I think... in any case, this validation is better than not having any validation at all... so I am OK to merge it as is and open an issue to keep investigating the behavior.